draft-ietf-rats-reference-interaction-models.md

```diff
@@ -547,15 +563,39 @@ Initiation by the Attester can result in unsolicited pushes of Evidence to the V
 Initiation by the Verifier always results in solicited pushes to the Verifier.
 
 The Uni-Directional model uses the same information elements as the Challenge/Response model.
-In the sequence diagram above, the Attester initiates the conveyance of Evidence (comparable with a RESTful POST operation or the emission of a beacon).
+In the sequence diagram above, the Attester initiates the conveyance of Evidence (comparable with a RESTful POST operation).
 While a request of Evidence from the Verifier would result in a sequence diagram more similar to the Challenge/Response model (comparable with a RESTful GET operation).
-The specific manner how Handles are created and used always remains as the distinguishing quality of this model.
+The specific manner how Handles are generated is not in scope of this document.
+One example of a specific handle representation is {{-epoch-markers}}.
 
 In the Uni-Directional model, handles are composed of cryptographically signed trusted timestamps as shown in {{-TUDA}}, potentially including other qualifying data.
-The Handles are created by an external 3rd entity -- the Handle Distributor -- which includes a trustworthy source of time, and takes on the role of a Time Stamping Authority (TSA, as initially defined in {{-TSA}}).
+The Handles are created by an external trusted third party (TTP) -- the Handle Distributor -- which includes a trustworthy source of time, and takes on the role of a Time Stamping Authority (TSA, as initially defined in {{-TSA}}).
 Timestamps created from local clocks (absolute clocks using a global timescale, as well as relative clocks, such as tick-counters) of Attesters and Verifiers MUST be cryptographically bound to fresh Handles received from the Handle Distributor.
 This binding provides a proof of synchronization that MUST be included in all produced Evidence.
-Correspondingly, conveyed Evidence in this model provides a proof that it was fresh at a certain point in time.
+Correspondingly, conveyed Evidence in this model provides a proof that it was fresh at the specific point in time when the handle was generated during the Handle Generation phase.
```

Comment:
```suggestion
This model provides proof that Evidence generation happened after the Handle generation phase.
```

Reply:
@henkbirkholz I have added another small bit to account for "the other side."